tasks/section4.yml

```diff
@@ -390,6 +453,29 @@
       - patch
       - rule_4.2.1.3
 
+- name: "SCORED | 4.2.1.3 | PATCH | Ensure rsyslog default file permissions configured"
+  file:
+      path: "{{ item }}"
+      mode: 0600
+      state: touch
```

Comment:
Would it make sense to gate this behind a workaround var?

Reply:
yep, will do.